src/compute/src/render/top_k.rs

```diff
@@ -322,28 +332,26 @@ where
                 }
             });
 
+            let debug_name = debug_name.to_string();
+            let (partial, errs) = collection.ensure_monotonic(move |data, diff| {
+                warn!("[customer-data] MonotonicTop1 expected monotonic input but received {data:?} with diff {diff:?} in dataflow {debug_name}");
```

Comment:
Could you add a newline here? I have the slight suspicion that long strings tend to break rustfmt. It sees them, decides that it can't do anything about them and just gets depressed and gives up.

Reply:
I fully expected rustfmt to deal with this for me, woops.

Reply:
I was discussing with @vmarcos more generally this pattern of warn/error, and will file an issue around clarifying it and the exposure of customer data, once I've figured out the right phrasing, but it did strike me that we're probably going to do this a lot, and it might make sense to have a macro that has the
```
foo!(data=?data, ..., "message")
```
form since it could more easily do the `error!` without the data, and the `warn!` with.